Assets/Mirror/Editor/Weaver/Extensions.cs

```diff
@@ -112,5 +112,26 @@ public static bool CanBeResolved(this TypeReference parent)
             }
             return true;
         }
+
+
+        public static MethodReference MakeHostInstanceGeneric(this MethodReference self, GenericInstanceType instanceType)
```

Comment:
a comment about what this does might be useful

Reply:
Added comment explanation